ethereum/scripts/sendEth.js

```diff
@@ -1,53 +1,63 @@
 // Simple script for sending transactions to a locally deployed Bank contract
 module.exports = async () => {
     const Web3 = require("web3");
-  
+
     // Contract abstraction
     const truffleContract = require("@truffle/contract");
     const contract = truffleContract(
-      require("../build/contracts/Bank.json")
+      require("../build/contracts/EthereumApp.json")
     );
 
-    const RECIPIENT = Buffer.from(
-       "8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48", "hex"
-    );
-  
-    const AMOUNT = 10;
+    // Parameters
+    const ethAmountStr = process.argv[4].toString();
+    if (!ethAmountStr) {
+        console.log("Must provide an Ethereum amount")
+        return
+    }
+
+    const polkadotRecipient = process.argv[5].toString();
+    if (!polkadotRecipient) {
+      console.log("Must provide a Polkadot recipient")
+      return
+    }
+    const recipient = Buffer.from(polkadotRecipient, "hex");
 
     let provider = new Web3.providers.HttpProvider("http://localhost:7545");
-  
     const web3 = new Web3(provider);
     contract.setProvider(web3.currentProvider);
 
+    const AMOUNT = web3.utils.toWei("10");
```

Comment:
Is `AMOUNT` used anywhere?

Reply:
Ah, this is likely leftover from when the function parameters were hardcoded.